File: fizzbuzz.rb
for i in 1..100 do
    if i % 15 == 0 then
        puts "FizzBuzz"
    elsif i % 5 == 0 then
        puts "Buzz"
    elsif i % 3 == 0 then
        puts "Fizz"
    else
        puts i
    end
end

File: fizzbuzz.pl
for($i=1; $i <= 100; ++$i)
{
    if($i % 15 == 0) {
        print "FizzBuzz\n";
    } elsif($i % 3 == 0) {
        print "Fizz\n";
    } elsif($i % 5 == 0) {
        print "Buzz\n";
    } else {
        print "$i\n";
    }
}
